### PR TITLE
styling improvement tweaks

### PR DESCRIPTION
**Pull request by ammarhaiderbjss**
## Description and relevant Jira numbers
- Removes the typo “Z” in front of the test requirements section title
- Adds a little spacing between the D/F and the circular counter
- Centers the tick icon for manoeuvres and test requirements when running on ios
- Aligns the toolbar elements with the elements on the rest of the page
- Makes the overlay slightly darker and quicker to appear

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [ ] PR has an understandable description

## Git feature branch checklist:

- [ ] branch name comply with our branching strategy
- [ ] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [ ] PR link added to JIRA
- [ ] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

### CODE REVIEW

**Comment by ammarhaiderbjss** [comment]
IMG_594C241C3EF3-1

File: src/pages/test-report/components/toolbar/toolbar.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 toolbar {
   #wrapper {
     height: 68px;
-    padding: 12px;
+    padding: 12px 24px 12px 24px;
```
Comment (johnmorleybjss) [inline src/pages/test-report/components/toolbar/toolbar.scss]: You could use vertical/horizontal shorthand here.

**Comment by Jason-Pullen** [comment]
on the screenshot the test requirements tick indicators are just a tiny bit high.

File: src/pages/test-report/components/toolbar/toolbar.scss (hunk 1 above)
Comment (ammarhaiderbjss) [reply src/pages/test-report/components/toolbar/toolbar.scss]: true, done

**Pull request merged**